### DISCUSSION TITLE
Where are the good 'ol packages?

### DISCUSSION THREAD

**Discussion by trusktr** [pulsar-edit/.github#143] [Pulsar Support] [answered]
For example, I searched for "vim mode" and there are no results. I was looking for https://github.com/t9md/atom-vim-mode-plus (the best implementation of Vim to ever exist :) )

Edit: Hmm, something is wrong with my install. I do see it here:

https://web.pulsar-edit.dev/packages/vim-mode-plus

Just not when I search inside of Pulsar.

![PXL_20230227_203407153](https://user-images.githubusercontent.com/297678/221679356-b72968b2-9d9b-4757-9616-28dc243e69e7.jpg)

The install button from the website isn't working either. It tries to open in Pulsar, but says

```
Failed to load vim-mode-plus - try again later.
```

**Comment by Spiker985** [comment, marked as answer]
What does your output show? Normally this is related to being unable to access api.pulsar-edit.dev, or something else

**Reply by Spiker985** [reply]
Something else to note - this is really something that is an `issue`, not a discussion

**Reply by trusktr** [reply]
I think maybe I was having network issues I was not aware of. I can see vim-mode-plus now in the "Install" section:

<img width="1187" alt="Screenshot 2023-03-06 at 3 00 22 PM" src="https://user-images.githubusercontent.com/297678/223276313-0b70537d-85fd-432c-b426-be13bff7d621.png">

I still don't see it in the "Packages" section though:

<img width="1101" alt="Screenshot 2023-03-06 at 3 00 04 PM" src="https://user-images.githubusercontent.com/297678/223276347-d3b20b26-187c-4dd7-a05a-00a429f8310d.png">

I think I expected "Packages" to be for searching for new packages, but I realized this is only for seeing installed packages. Maybe the UX can be combined into one so there's only a "Packages" section, and no more separate "Install" section.

**Reply by Daeraxa** [reply]
I do know what you mean as I've fallen for that a few times in the past. The main difference is that the `Install` section only shows stuff that it grabs from the package backend whereas `Packages` allows you to manage *all* packages, the ones downloaded from the backend, core, dev and git (downloaded from git repos).

I think thats something we need more discussion around if we choose to do anything with it.

**Comment by Daeraxa** [comment]
What is the failure in the failure message?

**Comment by Daeraxa** [comment]
As for the failure to find it, searching for `vim-mode` should find it. ~~I'm sure I remember something about fuzzy finding not being available in-app yet (or I'm talking rubbish).~~ Ignore this, its the same as on the website

**Reply by savetheclocktower** [reply]
Searching for `vim-mode-plus` works for me, but not for @trusktr, so something else is going on.

**Reply by confused-Techie** [reply]
@savetheclocktower It could be rate limiting.

But my initial knee jerk assumption. They took a photo with their phone, so I'm assuming this is a company computer. This could be the case of a corporate network that is using SSL decryption, where PPM won't trust the cert, so they have to explicitly disable the SSL checking. It's an option listed on PPM's readme. But without knowing any actual error it's really hard to say

**Reply by Spiker985** [reply]
I've had an issue on a work machine implementing Cisco Umbrella, acting as a middleman for certs - which doesn't apply the certs across the board, only to certain applications like Chrome

In other applications, like Postman, you can get around it by providing the Certificate Authority cert
